numpy/ma/tests/test_core.py

```diff
@@ -447,6 +447,13 @@ def test_deepcopy(self):
         assert_equal(copied.mask, [0, 0, 0])
         assert_equal(a.mask, [0, 1, 0])
 
+    def test_fstring(self):
```

Comment:
I think perhaps better to just call this `test_format`? f-strings are a distraction here, what you're actually testing is the `format()` builtin

Reply:
Ahh, this is a good point. i agree. Thanks!